packages/edit-post/src/components/header/template-title/index.js

```diff
@@ -33,12 +41,33 @@ function TemplateTitle() {
 	}
 
 	return (
-		<span className="edit-post-template-title">
-			{
-				/* translators: 1: Template name. */
-				sprintf( __( 'Editing template: %s' ), templateTitle )
-			}
-		</span>
+		<Dropdown
+			position="bottom center"
+			className="edit-post-template-top-area"
+			contentClassName="edit-post-template-top-area__popover"
+			renderToggle={ ( { onToggle } ) => (
+				<>
+					<div className="edit-post-template-title">
+						{ __( 'About' ) }
+					</div>
+					<Button isSmall isTertiary onClick={ onToggle }>
```

Comment:
```suggestion
					<Button isSmall isTertiary onClick={ onToggle } aria-label={ __( 'Template Options' ) }>
```

Reply:
Agree an aria-label would be good because for a screenreader there's no context about what this does.

Ideally an aria-label would read the template title as well, because the 'Template Options' label will override the button text.

The button should also implement `aria-haspopup` and `aria-expanded`.